capture-sdk/sdk/src/main/java/net/gini/android/capture/GiniCaptureFragment.kt

```diff
@@ -294,10 +297,15 @@ class CaptureFragmentFactory(
     override fun instantiate(classLoader: ClassLoader, className: String): Fragment {
         when (className) {
             CameraFragment::class.java.name -> return CameraFragment().apply {
+                val options = BarcodeScannerOptions.Builder()
+                    .setBarcodeFormats(Barcode.FORMAT_QR_CODE)
+                    .build()
+
                 setListener(
                     cameraListener
                 )
                 setCancelListener(cancelListener)
+                setBarcodeScanner(BarcodeScanning.getClient(options))
```

Comment:
[Clarification] Why do we need to pass the barcode scanner here?

Previously we encapsulated the MLKit dependency in `QRCodeDetectorTaskMLKit()` so that we can swap it out with some other QR code detector in the future.

I would prefer if we could avoid exposing MLKit BarcodeScanner.

Reply:
I found a way to avoid exposing MLKit by making the `CameraFragmentImpl.initQRCodeReader()` method protected and then overriding it in a subclass in the tests to do nothing.

Let me know please if that is ok for you.